matlab_simulation/05-estimation/kalman_filter/kf_example1.m

```diff
@@ -0,0 +1,133 @@
+% Kalman Filter Example 1: Temperature Control
+% 'Estimation 1' topic slide 48
+% State : Current temperature difference with outside
+% Motion : Decaying temperature + furnace input + disturbances
+% Measurement : Current temperature difference
+% Controller : Bang bang controller
+
+clear all
+clc
+close all
+
+% Example number
+example = 1;
+
+% Discrete time step
+dt = 0.1;
+
+% Prior
+mu = 10; % mean (mu)
+S = 1;   % covariance (Sigma)
+
+% Get matrices for motion model and measurement model
+[A,B,R,n] = motion_model(example,dt);
+[C,D,Q,m] = measurement_model(example);
+
+% Store in a structure
+ssm.A = A;
+ssm.B = B;
+ssm.C = C;
+ssm.D = D;
+ssm.R = R;
+ssm.Q = Q;
+ssm.n = n;
+ssm.m = m;
+
+% Simulation Initializations
+Tf = 3;
+T = 0:dt:Tf;
+x = zeros(1,length(T)+1);
+x(1) = mu+sqrt(S)*randn(1);
```

Comment:
I left the disturbance in the first state as it was present in Professor Waslanders original code. I guess the example states that we might no have a good prior and hence the disturbance.

Sure, I can replace randn(1) with randn, but I think it makes for slightly better readability with the randn(1).